src/app/models/constraint-models/concept-type.ts

```diff
@@ -8,6 +8,7 @@
 
 export enum ConceptType {
   CATEGORICAL = 'CATEGORICAL',
+  CATEGORICAL_OPTION = 'CATEGORICAL_OPTION',
```

Comment:
I thought 'CATEGORICAL_OPTION' would be deprecated, i.e. a categorical concept is not divisible.

Reply:
I am not sure about the CATEGORICAL_OPTION, I just noticed that even if it is not a part of this enum, it is showed in variables panel. I will ask about it.

Reply:
CATEGORICAL_OPTION and TEXT types will be removed from variables panel (see [TMT-627](https://jira.thehyve.nl/browse/TMT-627))